samples/ClassroomLabs/Scripts/BringImageToSharedImageGallery/Readme.md

```diff
@@ -0,0 +1,31 @@
+# Introduction
+This script is used to bring a custom image from an Azure virtual machine (VM) to shared image gallery.
+
+## Notes
+- The script must be executed using administrator privileges.
+
+# Directions
+1. Open a PowerShell window.  Make sure that the window notes it is running under administrator privileges.
```

Comment:
Is running as admin required?  I typically don't run as admin and haven't had trouble with it?

Reply:
I just copied this from other scripts ReadMes...no, not required...will remove.